scripts/gulp/tasks/docs.ts

```diff
@@ -212,14 +203,26 @@ task('serve:docs:stop', () => forceClose(server));
 // Watch
 // ----------------------------------------
 
-task('watch:docs:component-info', cb => {
-  // rebuild component info
-  watch(componentsSrc, series('build:docs:component-info'))
-    .on('add', logWatchAdd)
-    .on('change', logWatchChange)
-    .on('unlink', logWatchUnlink);
-
-  cb();
+task('watch:docs:component-info', () => {
+  Object.values(getAllPackageInfo()).forEach(pkg => {
+    if (pkg.packageJson.gulp?.componentInfo) {
+      const internalTask = () =>
+        src(pkg.packageJson.gulp?.componentInfo, { cwd: pkg.packagePath })
+          .pipe(
+            cacheNonCi(gulpReactDocgen(paths.docs('tsconfig.json'), ['DOMAttributes', 'HTMLAttributes']), {
+              name: 'componentInfo-3',
+            }),
+          )
+          .pipe(dest('componentInfo', { cwd: pkg.packagePath }));
+      // @ts-ignore
+      internalTask.displayName = 'build:docs:component-info';
+
+      const watcher = watch(pkg.packageJson.gulp?.componentInfo, { cwd: pkg.packagePath }, internalTask);
+
+      watcher.on('add', logWatchAdd);
+      watcher.on('change', logWatchChange);
+    }
```

Comment:
It will be great to remove this, but running three copies of `watch:component-info` explodes my machine 😭